### PR TITLE
Just a Small Typo fix

### PR DESCRIPTION
**Pull request by roswellgames**
**Pull Request Description**

_Please include a general description of the changes made, why they were necessary, 
and any other general things to note._

**Pull Request Checklist**:
* [ ] Have you followed the guidelines in our Contributing document and Code of Conduct?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/project-error/npwd/pulls) for the same update/change?
* [ ] Have you built and tested NPWD in-game after the relevant change?

### CODE REVIEW

**Comment by TasoOneAsia** [comment]
Appreciate the PR! Also, I think there may also be other typos strewn around documentation (that I saw). If you were interested, in also trying to resolve those it would be heavily appreciated. If not, I can merge this PR.

**Comment by roswellgames** [comment]
Yea sure ill go through and have a look whilst i'm reading through :)

**Comment by itschip** [comment]
> Appreciate the PR! Also, I think there may also be other typos strewn around documentation (that I saw). If you were interested, in also trying to resolve those it would be heavily appreciated. If not, I can merge this PR.

You don't say. This poor soul will be trapped in docs typos.

**Comment by roswellgames** [comment]
Small Turn Big Typo Update :)

File: docs/npwd/Roadmap.md [hunk 1]
```diff
@@ -33,15 +33,15 @@ These are features we plan on adding to the phone, but currently have no set tim
     - Will use [NGWD](https://github.com/project-error/new-garage-who-dis) as this is planned to be fully standalone. 
 - [ ] GPS System	
     - Send current location.
-    - Intervally update real time position.
+    - Intervalley update real time position.
```
Comment (RockySouthpaw) [inline docs/npwd/Roadmap.md:36]:
```suggestion
    - Update real time position within a specified time frame.
```

File: docs/npwd/faq.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ This can sometimes happen after we change the structure of our client stored set
 
 ### Event playerJoining was not safe for net
 
-You need to update your artifacts to a version beyond `3622`. Don’t know your artifact version? Type `version` into your console. 
+You need to update your artefacts to a version beyond `3622`. Don’t know your artefact version? Type `version` into your console.
```
Comment (RockySouthpaw) [inline docs/npwd/faq.md:26]: This is actually spelled correctly already. As per the url here https://runtime.fivem.net/artifacts/fivem/build_proot_linux/master/

File: docs/npwd/start/Config.md [hunk 3]
```diff
@@ -5,11 +5,11 @@ sidebar_label: Advanced Configuration
 ---
 
 NPWD includes a `config.json` file in the root directory that allows for further customization of phone
-behavior. This file is what allows for us to offer framework-agnostic compatibility as well as for multitude
+behaviour. This file is what allows for us to offer framework-agnostic compatibility as well as for multitude
```
Comment (RockySouthpaw) [inline docs/npwd/start/Config.md:8]: This is also already spelled correctly.

File: docs/npwd/start/Install.md [hunk 4]
```diff
@@ -139,7 +139,7 @@ If you can't find your application you just created, check your settings pages [
 
 ### Setting up Discord Log Integration
 
-NPWD can integrate with Discord to log certain actions. As of release v1.0.3, the only use case is to log reported tweets and marketplace listings. This requires a webhook to be setup and the following convar
+NPWD can integrate with Discord to log certain actions. As of release v1.0.3, the only use case is to log reported tweets and marketplace listings. This requires a web hook to be setup and the following convar
```
Comment (RockySouthpaw) [inline docs/npwd/start/Install.md:142]:
```suggestion
NPWD can integrate with Discord to log certain actions. As of release v1.0.3, the only use case is to log reported tweets and marketplace listings. This requires a webhook to be setup and the following convar
```

File: docs/npwd/start/Intro.md [hunk 5]
```diff
@@ -40,7 +40,7 @@ to ease and streamline the phone development process.
 ## Standalone Information & Installation
 
 We have designed *NPWD* to be generally framework agnostic, meaning that it can easily be 
-integrated with both popular open source frameworks like ESX & QBCore or any arbitary 
+integrated with both popular open source frameworks like ESX & QBCore or any arborators
```
Comment (RockySouthpaw) [inline docs/npwd/start/Intro.md:43]:
```suggestion
integrated with both popular open source frameworks like ESX & QBCore or any arbitrary 
```

File: docs/npwd/start/Intro.md [hunk 6]
```diff
@@ -61,23 +61,23 @@ Please open an issue/enhancement on our Github [repository](https://github.com/p
   - 0.01 ms on idle and 0.05 while in use.
 - [Twitter](https://i.imgur.com/BjwovRR.png)
   - Like, reply, retweet, report and delete your own Tweets.
-  - Send emojis and images directly from the phone's gallery, or from an external url. *Gifs too!*
-  - NPWD features discord logging so all reported tweets will be sent to the configured webhook.
+  - Send emojis and images directly from the phone's gallery, or from an external URL. *Gifs too!*
+  - NPWD features discord logging so all reported tweets will be sent to the configured web hook.
```
Comment (RockySouthpaw) [inline docs/npwd/start/Intro.md:65]:
```suggestion
  - NPWD features discord logging so all reported tweets will be sent to the configured webhook.
```

File: docs/npwd/start/Intro.md [hunk 7]
```diff
@@ -86,7 +86,7 @@ Please open an issue/enhancement on our Github [repository](https://github.com/p
 - [Notes](https://i.imgur.com/0Hvvlah.png)
   - Something you want to remember in game? Make a note!
 - [Calculator](https://user-images.githubusercontent.com/55056068/147531020-b7527a69-0b0e-4e81-83c7-58ad836eab23.png)
-  - Peform calculations. 
+  - Reform calculations.
```
Comment (RockySouthpaw) [inline docs/npwd/start/Intro.md:89]:
```suggestion
  - Perform calculations. 
```

File: docs/npwd/start/Intro.md [hunk 8]
```diff
@@ -111,13 +111,13 @@ Please open an issue/enhancement on our Github [repository](https://github.com/p
...
-  - Choose betwen **twelve** languages as of v1.0.
+  - Choose between **twelve** languages as of v1.0.
   - Change frames, icon sets and themes.
   - Adjust Zoom (100% to 70%).
   - Filter notification preferences.
 - Discord Logging
-  - Follow our [documentation](../start/Install.md#setting-up-discord-log-integration) for intial setup.
-  - Never used a webhook before? Follow Discord's [documentation](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks) for creating a webhook.
+  - Follow our [documentation](../start/Install.md#setting-up-discord-log-integration) for initial setup.
+  - Never used a web hook before? Follow Discord's [documentation](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks) for creating a web hook.
```
Comment (RockySouthpaw) [inline docs/npwd/start/Intro.md:120]:
```suggestion
  - Never used a webhook before? Follow Discord's [documentation](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks) for creating a webhook.
```

**Review by RockySouthpaw — CHANGES_REQUESTED** [review]

**Comment by RockySouthpaw** [comment]
Some minor edits but otherwise fine.

File: docs/npwd/start/Config.md (hunk 3 above)
Comment (TasoOneAsia) [reply docs/npwd/start/Config.md:8]: Since this is a american/british English thing, I'll simply keep as was.

File: docs/npwd/faq.md (hunk 2 above)
Comment (TasoOneAsia) [reply docs/npwd/faq.md:26]:
```suggestion
You need to update your artifacts to a version beyond `3622`. Don’t know your artifact version? Type `version` into your console. 
```

File: docs/npwd/start/Config.md (hunk 3 above)
Comment (RockySouthpaw) [reply docs/npwd/start/Config.md:8]: Can we change color to colour

**Pull request merged**